queries/models/projeto_subsidio_sppo/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - projeto_subsidio_sppo
 
+## [9.2.7] - 2025-07-02
+
+### Adicionado
+
+- Adicionada coluna `feet_start_date` no modelo `viagem_planejada.sql` para o periodo anterior a `2024-04-01` (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/656)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)